### PR TITLE
NEW : payment on several expense report (SQL integration) 

### PR DESCRIPTION
**Pull request by atm-adrien**
This is the following PR of this one : https://github.com/Dolibarr/dolibarr/pull/23702

### CODE REVIEW

**Comment by atm-adrien** [comment]
@eldy

File: htdocs/install/mysql/tables/llx_paymentuser.sql [hunk 1]
```diff
@@ -16,14 +17,14 @@
 --
 -- ===================================================================
 
-create table llx_payment_expensereport
+create table llx_paymentuser
```
Comment (eldy) [inline htdocs/install/mysql/tables/llx_paymentuser.sql:20]: Why renaming this table. Current structure and usage seems ok.  ?

Comment (atm-adrien) [reply htdocs/install/mysql/tables/llx_paymentuser.sql]: Done

Comment (atm-adrien) [reply htdocs/install/mysql/tables/llx_paymentuser.sql]: @eldy

**Comment by atm-adrien** [comment]
@eldy

File: htdocs/install/mysql/tables/llx_payment_expense_report.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+-- ===================================================================
+
+create table llx_expensereport_payment_expensereport
```
Comment (eldy) [inline htdocs/install/mysql/tables/llx_payment_expense_report.sql:20]: Can you remind me the goal of feature:

Being able to pay a common expense report and split payment into n users sharing the same expense ?
If this is the need, why not just adding the field  "fk_paiementuser"  into table llx_payment_expensereport  ?
An expense report can alreaydy be splitted into several payments. So we can store the detail of each payment 

Or being able to make 1 payment for n expense reports of he same user ?

Comment (ludovicandrieux) [reply htdocs/install/mysql/tables/llx_payment_expense_report.sql:20]: Hi @eldy 

The goal is being able to make 1 payment for n expense reports of the same user.

**Pull request closed without merging**